tests/test_io.py

```diff
@@ -294,3 +294,66 @@ def test_load_neo_file_nwb():
             for trace in traces:
                 assert trace['stim_start'] == [250]
                 assert trace['stim_end'] == [600]
+
+
+@pytest.fixture
+def json_filename():
+    filename = 'tmp.json'
```

Comment:
Could you add those paths to the .gitignore?

Reply:
I think the fixture will take care of deleting the file once the test is done, no?

Reply:
Oh sorry yes. It's ok then